stytch/consumer/models/crypto_wallets.py

```diff
@@ -22,17 +22,17 @@ class SIWEParams(pydantic.BaseModel):
       - domain: Only required if `siwe_params` is passed. The domain that is requesting the crypto wallet signature. Must be an RFC 3986 authority.
       - uri: Only required if `siwe_params` is passed. An RFC 3986 URI referring to the resource that is the subject of the signing.
       - resources:  A list of information or references to information the user wishes to have resolved as part of authentication. Every resource must be an RFC 3986 URI.
-      - chain_id: The EIP-155 Chain ID to which the session is bound. Defaults to 1.
-      - statement: A human-readable ASCII assertion that the user will sign.
+      - chain_id: The EIP-155 Chain ID to which the session is bound. Defaults to 1. Must be the string representation of an integer between 1 and 9,223,372,036,854,775,771, inclusive.
+      - statement: A human-readable ASCII assertion that the user will sign. The statement may only include reserved, unreserved, or space characters according to RFC 3986 definitions, and must not contain other forms of whitespace such as newlines, tabs, and carriage returns.
       - issued_at: The time when the message was generated. Defaults to the current time. All timestamps in our API conform to the RFC 3339 standard and are expressed in UTC, e.g. `2021-12-29T12:33:09Z`.
       - not_before: The time when the signed authentication message will become valid. Defaults to the current time. All timestamps in our API conform to the RFC 3339 standard and are expressed in UTC, e.g. `2021-12-29T12:33:09Z`.
-      - message_request_id: A system-specific identifier that may be used to uniquely refer to the sign-in request.
+      - message_request_id: A system-specific identifier that may be used to uniquely refer to the sign-in request. The `message_request_id` must be a valid pchar according to RFC 3986 definitions.
     """  # noqa
 
     domain: str
     uri: str
     resources: List[str]
-    chain_id: Optional[int] = None
+    chain_id: Optional[str] = None
```

Comment:
Oooooh, what happened here? This is technically a breaking change.

Reply:
I see it's related to SIWE... has this been released yet?

Reply:
Jenn was talking about this [here](https://stytchio.slack.com/archives/C04LCF8GJQ6/p1723145031432859)

Reply:
If it's unreleased, this is fine as a MINOR update

Reply:
Yup it's unreleased

Reply:
Let's release the SIWE stuff first and I'll rebase off of that